Tmain/lang-sequel.d/args.ctags

```diff
@@ -3,3 +3,4 @@
 --_sequel-UnknownX={{
     (bye) ==
 }}
+-o -
```

Comment:
Good catch!

Could you write this in run.sh, not here.

Reply:
> Could you write this in run.sh, not here.

Sure.  I've done.